upgrade/rendering-changes/7-0-0.md

```diff
@@ -66,6 +67,10 @@ The `Open`, `Download` and `Print` tools have been moved to a submenu toggled on
 3. The scroll buttons (prev & next) now render the `k-tabstrip-prev` and `k-tabstrip-next` classes, respectively.
 4. The `k-pos-relative` class is no longer applied to the `<div class="k-tabstrip">` element. The component now receives an inline `position: relative` style instead.
 
+## TileLayout
+
+The tile elements (`<div class="k-tilelayout-item">`) render `tabindex="-1"` instead of `tabindex="0"` when the `Navigable` parameter of the TileLayout is `false`, which is by default.
```

Comment:
The `which is by default` bit could be somewhat ambiguous. 
```suggestion
The tile elements (`<div class="k-tilelayout-item">`) render `tabindex="-1"` instead of `tabindex="0"` when the `Navigable` parameter of the TileLayout is `false`, which is the default setting.
```